CompetitionTemplate/src/competition/subsystems/hanger/winch_commands/WinchStopCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.hanger.winch_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.hanger.WinchSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class WinchStopCommand extends BaseCommand{
```

Comment:
I'm assuming you want these stop commands to be the defaults on these subsystems?